com.unity.mobile.notifications/Runtime/iOS/AuthorizationRequest.cs

```diff
@@ -68,26 +67,53 @@ internal struct iOSAuthorizationRequestData
     /// </example>
     public class AuthorizationRequest : IDisposable
     {
+        bool _IsFinished;
```

Comment:
code style, private variable should be prefixed by m_

Reply:
Renamed.